data/AZ/programs.json

```diff
@@ -294,5 +294,13 @@
     "url": {
       "en": "https://www.garkaneenergy.com/energy-efficient-rebates"
     }
+  },
+  "az_arizonaPublicService_homePerformanceProgram": {
+    "name": {
+      "en": "Home Performance Program"
```

Comment:
Again not blocking, but because these program names currently get displayed standalone in the calculator I think for clarity it might be good to put APS or Arizona Public Service in here? I think we are not consistent about this today, maybe worth discussing as a team (the alternative would be to display authority somehow in the calculator, wider Design question?)

Reply:
That makes sense. I added it to our core sync agenda for the next meeting